src/pages/patientView/mutation/MutationInformationContainer.tsx

```diff
@@ -33,12 +89,34 @@ export default class MutationInformationContainer extends React.Component<IMutat
             },
             cosmic: {
                 name: "COSMIC"
-            }
+            },
+            alleleFreq: {
```

Comment:
using `tumorAlleleFreq` instead of `alleleFreq` would override the default value.